legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/domain/api/review/ReviewApi.java

```diff
@@ -50,6 +52,23 @@
      */
     List<Review> getReviews(String projectId, ReviewState state, Iterable<String> revisionIds, Instant since, Instant until, Integer limit);
 
+    /**
+     * Get all reviews with the given state.
+     * If state is null, all reviews are returned. Results are undefined for state {@link ReviewState#UNKNOWN}.
+     * remove duplicates to form a single list of reviews to return (with other constraints applied on top of that).
+     * Time filter range (since/until) is inclusive.
+     * If the limit equals to 0, effectively no limit is applied.
+     * if no projectType is provided the default mode of the system would be selected
+     * @param projectTypes the project type for which the reviews would be returned
+     * @param assignedToMe Boolean true / false the default is true is nothing is selected
+     * @param state       review state
+     * @param since       this time limit is interpreted based on the chosen state, for example: if only committed reviews are fetched, 'since' will concern the commited time
+     * @param until       this time limit is interpreted based on the chosen state, for example: if only committed reviews are fetched, 'since' will concern the commited time
+     * @param limit       maximum number of reviews to get
+     * @return reviews
+     */
+    List<Review> getReviews(Set<ProjectType> projectTypes, Boolean assignedToMe, ReviewState state, Instant since, Instant until, Integer limit);
```

Comment:
Let's make assignedToMe a boolean, instead of a Boolean, and handle the default case at the resource level.